### DISCUSSION TITLE
Desktop

### DISCUSSION THREAD

**Discussion by Tolcan** [devmfc/debian-on-amlogic#59] [Q&A]
Hi Thanks for the great job. 
I just would like to give a feedback;
On latest Debian version, everything works fine 'Headless'.
Your desktop env installed by the script and my tasksel installed xfce and gnome have the same problem: Networking works with root. Networking does not work with the script created user... 
Device: Beelink GT-King
Thank you so much again

**Comment by devmfc** [comment]
Thanks.

But I could not reproduce the problem, booted a clean Noble image, installed Gnome desktop with my script and the normal user and could browse the internet?

**Reply by Tolcan** [reply]
Hi again,
Thank you for your reply.
I also tried the Ubuntu release which has no problem. 
On debian, I have installed xfce via tasksel once again which has the same problems.
Nevertheless, headless Debian is just perfect. I installed it on mmc, it works great.
Thank you so much for your effort..
Kind regards,
